templates/semantic-ui/invenio_app_rdm/header_frontpage.html

```diff
@@ -58,7 +58,8 @@
       {%- block navbar %}
         {{ super() }}
         {%- block site_banner %}
-          {{ super() }}
+          {% from 'invenio_banners/banner.html' import banner %}
+          {{ banner(ui_classes="") }}
```

Comment:
why the `ui_classes=""`? what is the drawback?

Reply:
please check https://github.com/inveniosoftware/invenio-banners/pull/43
by default someone hardcoded margin and other classes in python, I don't want to break it for everyone in RDM (anyone who has banner at the top), so I made it configurable in the macro, instead of hardcoding it. Here I just remove the unwanted classes by setting them to empty